### ISSUE TITLE
Instructor students page: accessibility issue for course summary headers

### ISSUE THREAD

**Issue by weiquu** [TEAMMATES/teammates#12333]
Currently, the summary headers for the course in the instructor students page (`To instructor pages` -> `Students` tab) are being read separately by the screen reader. For example, the below headers would be read as "9" then "students", "2" then "sections", and "3" then "teams" by the screen reader:
<img width="1129" alt="headers" src="https://user-images.githubusercontent.com/48304907/229411531-74718477-a626-43cc-86a1-63409552729b.png">

This behaviour is confusing for those using screen readers. The expected behaviour should be for the screen reader to read the summary text together (e.g. "9 students", "2 sections", and "3 teams").

**Comment by athakaras** [comment]
Hello! I am a first time contributor and I would love to take on this issue. Can I please work on that?

**Comment by domlimm** [comment]
@athakaras Hello! Feel free to submit a PR for this issue 👍🏻

**Comment by athakaras** [comment]
do you know which file this issue is reffering to? because i'm having difficulty in finding it..

**Comment by rhyses-pieces** [comment]
Hello! I took a look around the codebase and I was able to get a gist of what's going on: in the `instructor-student-list-page` component, the number is in a separate heading tag than the rest of the text. The issue looks like this for students, and so forth:

```html
<h3 class="card-title inline">{{courseTab.stats.numOfStudents}}</h3>
<h4 class="card-text inline"> students</h4>
```

My proposed solution for this issue would be to simply move the text (e.g, `students`, `sections`, `teams`) into the same heading tag:

```html
<h3 class="card-title inline">{{courseTab.stats.numOfStudents}} students</h3>
```

I already forked the repo and cloned it to my local system, but I wanted to see if there were other suggested fixes that could be made. So far as I can tell, it _seems_ like there's a specific emphasis on the numbers, but I'm not sure if that's an intentional part of the design.

I'm interested in working on this issue both in discussion and in code!

**Comment by zhaojj2209** [comment]
@rhyses-pieces Thank you for looking into this issue. Yes, the different headings were used so as to increase the font size of the text level. However, headings should never be used for styling purposes, and this should be changed. In fact, I would argue that headings should not be used at all here as these cards are not headings semantically.

In my opinion, the difference in font size is minimal enough that we can do without it altogether, so the simple fix here would be to just combine the text into one tag.

**Comment by rk22277** [comment]
@domlimm Hello ! I am interested to work on this...

**Comment by EuniceSim142** [comment]
Hello @rk22277, thanks for your interest! However since @athakaras has already shown interest in working it, I would suggest you look for another issue to contribute to.

**Comment by rk22277** [comment]
@EuniceSim142, thank you
